### ISSUE TITLE
Support for "powered" library actors

### ISSUE THREAD

**Issue by sorenhan** [microsoft/mixed-reality-extension-sdk#479]
objects with createfromlibrary can benefit from having a single value that can be set by the mre. It could be either binary or a floating point value, I'm not sure what is best.

While a whole Redstone-style power system is possible, and could be immensely powerful, I am explicitly not going there with this request - that would require a ton of planning.

**Comment by stevenvergenz** [comment]
Isn't what you're describing an actor-based RPC?

**Comment by sorenhan** [comment]
No, this would be much simpler than RPC - RPC is flexible, but requires knowledge about which RPC calls are available for which actor. This is a single variable that an object could use as it deems appropriate. For a non-Minecraft example that is more suitable, something like this old game's logic system: https://www.youtube.com/watch?v=1FOZvbiYrU8
